module-types/Cisco/DS-CAC-650W-I.yml

```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: DS-CAC-650W-I
+part_number: DS-CAC-650W-I
+weight:
+  - value: 2.42
+    unit: lb
+comments: '[Cisco MDS 9148T 32-Gbps 48-Port Fibre Channel Switch Data Sheet](https://www.cisco.com/c/en/us/products/collateral/storage-networking/mds-9148t-32-gbps-48-port-fibre-channel-switch/data-sheet-c78-740623.html)'
+power-ports:
+  - name: Input
```

Comment:
This will throw a duplicate errror.

I prefer: PSU{module} for most PSU's

Reply:
Modified the template file to replace "Input" with "PSU{module}".